frontend/src/containers/RegistrationContainer/RegistrationContainer.js

```diff
@@ -0,0 +1,48 @@
+import Registration from "../../components/Registration/Registration";
+import { ACCOUNTS } from "../../properties/endpoints";
+
+function postRegistrationDetails(registrationDetails) {
+	console.log(registrationDetails);
+	return fetch(ACCOUNTS, {
+		method: "POST",
+		headers: {
+			"Content-Type": "application/json",
+		},
+		body: JSON.stringify(registrationDetails),
+	}).then((response) => response.json());
+}
+
+function formatDetails(registrationDetails) {
+	const MONTH_ADJUSTER = 1;
+	return {
+		name: {
+			first: registrationDetails.firstName,
+			last: registrationDetails.lastName,
+		},
+		email: registrationDetails.email,
+		dateOfBirth: {
+			year: registrationDetails.dateOfBirth.getFullYear(),
+			month: registrationDetails.dateOfBirth.getMonth() + MONTH_ADJUSTER,
+			day: registrationDetails.dateOfBirth.getDate(),
+		},
+		username: registrationDetails.username,
+		password: registrationDetails.password,
+	};
+}
+
+function handleFormSubmit(registrationDetails, event) {
+	event.preventDefault();
+	event.target.reset();
+
+	const formattedRegistrationDetails = formatDetails(registrationDetails);
+
+	postRegistrationDetails(formattedRegistrationDetails)
+		.then(console.log)
```

Comment:
When I use the client I get some log output. Is this the reason? We want to avoid log output in "production"

Reply:
Yes, the use case states that the system displays a confirmation message, but that is not yet implemented. My idea is to use a [Snackbar](https://material-ui.com/components/snackbars/#snackbar) for that and any error message.